src/config/ConfigManager.cpp

```diff
@@ -534,6 +534,7 @@ CConfigManager::CConfigManager() {
     m_pConfig->addConfigValue("binds:movefocus_cycles_fullscreen", Hyprlang::INT{1});
     m_pConfig->addConfigValue("binds:disable_keybind_grabbing", Hyprlang::INT{0});
     m_pConfig->addConfigValue("binds:window_direction_monitor_fallback", Hyprlang::INT{1});
+    m_pConfig->addConfigValue("binds:pin_fullscreen", Hyprlang::INT{0});
```

Comment:
a name more like `allow_pin_fullscreen` makes more sense intuitively

+ missing documentation in configDescriptions.hpp